tensorflow/python/ops/image_ops.py

```diff
@@ -62,6 +62,7 @@
 @@non_max_suppression
 @@non_max_suppression_v2
 @@sample_distorted_bounding_box
+@@sample_distorted_bounding_box_v2
```

Comment:
In fact, it's ever worse: Because we're guaranteeing forward compatibility, this change has to be split in two: One that introduces the new _v2 kernel (hidden), and no less than three weeks later, one that switches the behavior of the python op from v1 to _v2 by hiding the v1 op and adding the python wrapper.